packages/react-native-reanimated/src/index.ts

```diff
@@ -287,6 +287,7 @@ export {
   ScreenTransition,
   startScreenTransition,
 } from './screenTransition';
+export { ReanimatedNativeView } from './specs';
```

Comment:
How about `ReanimatedView` instead?

Reply:
I suggested that name to be more explicit, but we can use just `ReanimatedView` instead 👍